.golangci.yml

```diff
@@ -0,0 +1,28 @@
+run:
+  deadline: 3m
+  modules-download-mode: readonly
+
+linters-settings:
+  gocyclo:
+    min-complexity: 20
+  goconst:
+    min-len: 3
+    min-occurrences: 5
```

Comment:
Remain the default value `3` is enough. 😢

Reply:
I will remove this goconst configuration.